tests/Earthfile

```diff
@@ -0,0 +1,27 @@
+VERSION 0.7
+
+FROM debian:stable-slim
+
+# BASH, CURL, GPG, NODEJS, NPM, JDK, JRE
+dependencies:
+    RUN apt-get update
+    RUN apt-get install bash
+    RUN apt-get install curl -y
+    RUN apt-get install gpg -y
+    RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+    RUN apt-get install nodejs -y
+    RUN apt-get install npm -y
+    RUN apt-get install default-jre -y
+
+generate-client:
+    FROM +dependencies
+    
+    # TODO: GET RIGHT FILE FROM POEM GENERATION / BY COPY FILE IN REPO
+    ARG openapispec_file="./petstore.yaml"
+    COPY $openapispec_file .
+
+    RUN npm install @openapitools/openapi-generator-cli -g
```

Comment:
Moving this line in the dependencies target will make it cache even when the openapispec_file changes

Reply:
Nice to know!
We can create a separate target just for this little sense to me to add this step to a dependency target for reusability.
When we get poem running these 2 first lines will be changed with the target we'll use to generate the openapi doc